content/en/blog/2025/announcing-etcd-3.6.md

```diff
@@ -0,0 +1,416 @@
+  - [Migration to v3store](#migration-to-v3store)
+  - [Downgrade](#downgrade)
+  - [Feature Gates](#feature-gates)
+  - [Livez/readyz checks](#livezreadyz-checks)
+  - [v3discovery](#v3discovery)
+- **[Breaking changes](#breaking-changes)**
+  - [Old Binaries Are Incompatible with New Schema Versions](#old-binaries-are-incompatible-with-new-schema-versions)
+  - [Peer Endpoints No Longer Serve Client Requests](#peer-endpoints-no-longer-serve-client-requests)
+  - [Clear boundary between etcdctl and etcdutl](#clear-boundary-between-etcdctl-and-etcdutl)
+- **[Critical bug fixes](#critical-bug-fixes)**
+- **[Upgrade issue](#upgrade-issue)**
+- **[Performance](#performance)**
+  - [Memory](#memory)
+  - [Throughput](#throughput)
+- **[Testing](#testing)**
+- **[Platforms](#platforms)**
+- **[Dependencies](#dependencies)**
+  - [Dependency Bumping Guide](#dependency-bumping-guide)
+  - [Core dependencies](#core-dependencies)
+  - [grpc-gateway@v2](#grpc-gatewayv2)
+  - [grpc-ecosystem/go-grpc-middleware/providers/prometheus](#grpc-ecosystemgo-grpc-middlewareprovidersprometheus)
+- **[Community](#community)**
+  - [etcd Becomes a Kubernetes SIG](#etcd-becomes-a-kubernetes-sig)
+  - [New Contributors, Maintainers, and Reviewers](#new-contributors-maintainers-and-reviewers)
+  - [New release team](#new-release-team)
+  - [Introducing the etcd Operator Working Group](#introducing-the-etcd-operator-working-group)
+- **[Future roadmaps](#future-roadmaps)**
+
+## Introduction
+
+Today, we are releasing [etcd v3.6.0][]. It has been nearly four years since the release of etcd v3.5.0 on June 15, 2021,
+making the arrival of etcd v3.6.0 a significant milestone. This release introduces several new features, makes significant
+progress on long-standing efforts like downgrade support and migration to v3store, and addresses numerous critical & major
+issues. Notably, it also brings major optimizations in memory usage, improving efficiency and performance.
+
+Beyond being a CNCF graduated project, etcd has also become a Kubernetes SIG (sig-etcd), marking a new chapter in its
+evolution. The ecosystem is expanding as well—one notable example is the formation of the working group etcd-operator,
+which promises to greatly improve the usability and operability of etcd.
+
+In this blog, we will highlight the most significant changes introduced in etcd v3.6.0 and discuss the roadmap for future
+development. For a detailed list of changes, please refer to the [CHANGELOG-3.6][].
+
+A heartfelt thank you to all the contributors who made this release possible!
+
+## Security
+
+etcd takes security seriously and has implemented several measures to enhance it. We have integrated `govulncheck` to
+scan the source code and `trivy` to scan container images, both as part of our workflow checks. Additionally,
+we continue to follow the [Security Release Process][] to ensure vulnerabilities are properly managed and addressed.
+
+## Features
+
+### Migration to v3store
```

Comment:
So this item is confusing a feature and a migration note.  I think we need to separate the two here and make them clear.  What are some of the advantages of moving to v3store?

Reply:
This is indeed a feature, which across multiple minor releases. 

Added advantage of v3store compared to v2store.

PTAL